File: ViewBase/Source/Others/CultureAwareBinding.vb
Imports System.Globalization
Imports System.Windows.Data

Public Class CultureAwareBinding
    Inherits Binding

    Sub New()
        MyBase.New()
        ConverterCulture = CultureInfo.CurrentCulture
    End Sub

    Sub New(path As String)
        MyBase.New(path)
        ConverterCulture = CultureInfo.CurrentCulture
    End Sub
End Class

File: CoreView/src/Others/CultureAwareBinding.cs
using System;
using System.Windows.Data;
using System.Globalization;

namespace Wells.CoreView
{
    public class CultureAwareBinding : Binding
    {
        public CultureAwareBinding() : base() { ConverterCulture = CultureInfo.CurrentCulture; }

        public CultureAwareBinding(string path) : base(path) { ConverterCulture = CultureInfo.CurrentCulture; }
    }
}
